src/main/java/uk/ac/sanger/sccp/stan/GraphQLMutation.java

```diff
@@ -582,6 +595,16 @@ public DataFetcher<OperationResult> reagentTransfer() {
         };
     }
 
+    public DataFetcher<RegisterResult> registerOriginalSamples() {
+        return dfe -> {
+            User user = checkUser(dfe, User.Role.normal);
```

Comment:
I hope 'checkUser' is checking whether the the user has given access right, in this case User.Role.normal. But, we only allow Admin users to register samples, so why checking it against normal ?

Reply:
Well spotted. I will change it to admin.